projects/igniteui-angular/src/lib/core/styles/themes/_palettes.scss

```diff
@@ -56,7 +56,7 @@ $default-palette: palette($primary: #09f, $secondary: #e41c77) !default;
 /// @access private
 $default-dark-palette: palette(
     $primary: color($default-palette, 'primary'),
-    $secondary: color($default-palette, 'secondary'),
+    $secondary: #72da67,
```

Comment:
Why did we change this color in the default palette?

Reply:
Because material uses the default palette and the color in the table is different from this one. All the other themes override it. If you want I can change it just for material?